stories/browser/src/UIPullerSheet.tsx

```diff
@@ -1,14 +1,37 @@
 import * as React from 'react';
-import { StyleSheet, useWindowDimensions, StatusBar } from 'react-native';
+import { useWindowDimensions, StatusBar } from 'react-native';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
 import { UIConstant } from '@tonlabs/uikit.core';
-import { UIBackgroundView, ColorVariants, useTheme } from '@tonlabs/uikit.themes';
-import { UIBottomSheet } from '@tonlabs/uikit.popups';
+import { UIBottomSheet, useIntrinsicSizeScrollView } from '@tonlabs/uikit.popups';
 import { ScrollView } from '@tonlabs/uikit.scrolls';
 
 const PULLER_HEIGHT = 12;
 
+function SheetContent({ children }: { children: React.ReactNode }) {
+    const insets = useSafeAreaInsets();
+    const { height } = useWindowDimensions();
+
+    const paddingBottom = Math.max(insets?.bottom || 0, UIConstant.contentOffset());
+    const { style: scrollIntrinsicStyle, onContentSizeChange } = useIntrinsicSizeScrollView();
+    return (
+        <ScrollView
+            bounces={false}
+            style={{
+                height: height - (StatusBar.currentHeight ?? 0) - insets.top - PULLER_HEIGHT,
```

Comment:
actually you don't need to calculate `height` when you use `useIntrinsicSizeScrollView` :)